sdk/keyvault/Microsoft.Azure.KeyVault.Core/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
+## 3.1.0-beta.1 (Unreleased)
```

Comment:
Any reason to change deprecated track 1 projects? We don't even package these anymore. I'll sign off because I see no harm, but it'd be nice to leave these untouched if possible.